frontend/shared/src/components/notification/Notification.tsx

```diff
@@ -1,8 +1,10 @@
-import { Notification } from 'hds-react';
+import { Notification as HDSNotification } from 'hds-react';
 import styled from 'styled-components';
 
-export const $Notification = styled(Notification)`
+const Notification = styled(HDSNotification)`
```

Comment:
In the end, it's still a styled component, why don't you keep it named `$Notification`?

Reply:
reverted